CMakeLists.txt

```diff
@@ -33,8 +33,11 @@ set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${CMAKE_SOURCE_DIR}/cmake")
 
 find_package(nlohmann_json 3.2.0 REQUIRED)
 
-find_package(xtensor-io 0.10.1 REQUIRED)
-message(STATUS "Found xtensor: ${xtensor_INCLUDE_DIRS}/xtensor")
+find_package(xtensor 0.23.0 REQUIRED)
+
+find_package(xtensor-io 0.12.0 REQUIRED)
+
+find_package(zarray 0.0.3 REQUIRED)
```

Comment:
I think it could be nice to set the version numbers into variables taht we can reuse in `zarrayConfig.cmake.in` so we have transitive dependencies.